packages/react-examples/src/react/TimePicker/TimePicker.Example.tsx

```diff
@@ -15,46 +19,72 @@ const timePickerStyles: Partial<IComboBoxStyles> = {
 
 export const TimePickerBasicExample: React.FC = () => {
+  const [datePickerDate, setDatePickerDate] = React.useState<Date>();
+  const [currentTimeString, setCurrentTimeString] = React.useState<string>('');
+
+  const onSelectDate = React.useCallback((selectedDate: Date) => {
+    setDatePickerDate(selectedDate);
+  }, []);
+
+  const onDateTimePickerChange = React.useCallback((time: Date) => {
+    setCurrentTimeString(time.toString());
+  }, []);
+
   const timeRange: ITimeRange = {
     start: 8,
     end: 14,
   };
 
   return (
-    <Stack tokens={stackTokens} styles={stackStyles}>
-      <TimePicker
-        styles={timePickerStyles}
-        useHour12
-        allowFreeform
-        autoComplete="on"
-        label={'TimePicker basic example'}
-        onChange={onChange}
-        defaultValue={new Date('November 25, 2021 09:15:00')}
-        useComboBoxAsMenuWidth
-      />
-      <TimePicker
-        styles={timePickerStyles}
-        showSeconds
-        allowFreeform
-        increments={15}
-        autoComplete="on"
-        label={'TimePicker with non default options'}
-        useComboBoxAsMenuWidth
-        timeRange={timeRange}
-        onChange={onChange}
-      />
-      <TimePicker
-        styles={timePickerStyles}
-        // eslint-disable-next-line react/jsx-no-bind
-        onFormatDate={onFormatDate}
-        useHour12
-        allowFreeform
-        autoComplete="on"
-        label={'TimePicker with custom time string'}
-      />
-    </Stack>
+    <>
+      <Stack tokens={stackTokens} styles={stackStyles}>
+        <TimePicker
+          styles={timePickerStyles}
+          useHour12
+          allowFreeform
+          autoComplete="on"
+          label={'TimePicker basic example'}
+          onTimeChange={onTimeChange}
+          currentDate={new Date('November 25, 2021 09:15:00')}
+          useComboBoxAsMenuWidth
+        />
+        <TimePicker
+          styles={timePickerStyles}
+          showSeconds
+          allowFreeform
+          increments={15}
+          autoComplete="on"
+          label={'TimePicker with non default options'}
+          useComboBoxAsMenuWidth
+          timeRange={timeRange}
+          onTimeChange={onTimeChange}
+        />
+        <TimePicker
+          styles={timePickerStyles}
+          // eslint-disable-next-line react/jsx-no-bind
+          onFormatDate={onFormatDate}
+          useHour12
+          allowFreeform
+          autoComplete="on"
+          label={'TimePicker with custom time string'}
+        />
+      </Stack>
+      <div style={{ paddingTop: '20px', maxWidth: '50%' }}>
+        <Label>{'DatePicker and TimePicker combination'}</Label>
+        <div style={{ display: 'grid', gridTemplateColumns: '1fr 1fr', gridColumnGap: '3px' }}>
+          <DatePicker
+            placeholder="Select a date..."
+            value={datePickerDate}
+            onSelectDate={onSelectDate}
+            minDate={new Date()}
```

Comment:
This feels like a bug. Every time the example re-renders (ex: when `onDateTimePickerChange` is called) `minDate` will be updated to whatever the current date is, meaning the value will tick forward in time based on interactions on the page.

I think you should define a constant for this that won't change based on React renders.

Reply:
This code snippet is no longer relevant and has been removed.